### PR TITLE
 deploy-chroot: Create kvm device instead of bind-mounting it.

### LINKED ISSUE #476: Clobbering of /dev/kvm permissions across deploy chroots on RHOMBI
When two people try to use deploy chroots at the same time, each of them will reassign the ownership of /dev/kvm to be root:kvm, so that users in the kvm (like the automatically-created one) will be able to run kvm-accelerated virtual machines.

Unfortunately, different deploy chroots may have different group IDs for the kvm group, especially if they have different operating system version numbers. This means they start to play "tug-of-war" over the ownership; whichever one doesn't have the right ownership will fail to launch VMs, and when the deploy chroot is restarted to fix it, it takes away the right ownership from the other deploy chroot.

This also has the side concern that it may change access on the host to a group that it shouldn't be giving access to -- like a group used to isolate vulnerable system services. While this isn't much of a concern on RHOMBI, which doesn't run very many services, it could be a problem in other deployment situations.

The reason they need to have the same ownership is that each chroot has a bind mount of /dev/kvm. We should just mknod (or maybe even cp) the node into each chroot, which will allow each deploy chroot to set its own ownership appropriately for its own device.

### PR DESCRIPTION
**Pull request by cryslith**
Fixes #476.

---

Checklist:

 - [x] I have split up this change into one or more appropriately-delineated commits.
 - [x] The first line of each commit is of the form "[component]: do something"
 - [x] I have written a complete, multi-line commit message for each commit.
 - [x] ~I have formatted any Go code that I have changed with gofmt.~
 - [x] ~I have written or updated appropriate documentation to cover this change.~
 - [x] ~I have confirmed that this change is covered by at least one appropriate test run by CI.~
 - [x] If my change includes new or modified functionality, I have tested that the changes work as expected.
 - [x] I have assigned this issue to an appropriate reviewer. (Choose @celskeggs if you are not otherwise certain.)
 - [x] I consider my PR complete and ready to be merged without my further input, assuming that it passes CI and code review.
 - [x] My changes have passed CI, including an automatic Jenkins deploy.
 - [x] My changes have passed code review.

### CODE REVIEW

File: deploy-chroot/enter-internal.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set -euo pipefail
 
 sudo mount -o rw,remount /proc/sys
+sudo mknod --mode='ug=rw,o=' /dev/kvm c 10 "$(grep -w kvm /proc/misc | cut -f 1 -d ' ')"
```
Comment (celskeggs) [inline deploy-chroot/enter-internal.sh]: How is this going to behave if kvm support happens to be missing?

Comment (cryslith) [reply deploy-chroot/enter-internal.sh]: poorly.

File: deploy-chroot/enter-internal.sh [hunk 2]
```diff
@@ -3,7 +3,9 @@
 set -euo pipefail
 
 sudo mount -o rw,remount /proc/sys
-sudo mknod --mode='ug=rw,o=' /dev/kvm c 10 "$(grep -w kvm /proc/misc | cut -f 1 -d ' ')"
+if grep -wq kvm /proc/misc; then
+    sudo mknod --mode='ug=rw,o=' /dev/kvm c 10 "$(grep -w kvm /proc/misc | cut -f 1 -d ' ')"
+fi
 sudo chgrp kvm /dev/kvm
```
Comment (celskeggs) [inline deploy-chroot/enter-internal.sh]: Maybe stick the chgrp in the if?
Might also be worth explicitly warning that it couldn't create the device node?

Comment (cryslith) [reply deploy-chroot/enter-internal.sh]: wooops

**Pull request merged**